Assets/BossRoom/Scripts/Client/UI/Lobby/CreateLobbyUI.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Text;
+using BossRoom.Scripts.Shared.Infrastructure;
+using Unity.Multiplayer.Samples.BossRoom;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BossRoom.Scripts.Client.UI
+{
+    public class CreateLobbyUI : MonoBehaviour
```

Comment:
Verbs in class names can be misleading (this could be interpreted as a command pattern :P -- public class DoSomethingCommand {} )
"LobbyCreationUI" would be clearer for example.

Reply:
Same goes for JoinLobbyUI.